test/protobuf/conformance_regressions_test.exs

```diff
@@ -83,6 +83,12 @@ defmodule Protobuf.ConformanceRegressionsTest do
 
     @describetag message_type: "protobuf_test_messages.proto3.TestAllTypesProto3"
 
+    test "map i32 i32" do
+      mod = ProtobufTestMessages.Proto2.TestAllTypesProto2
+      problematic_payload = <<194, 3, 0>>
+      assert m = %{map_int32_int32: %{0 => 0}} = mod.decode(problematic_payload)
```

Comment:
Without my change, the problematic map decodes to `%{nil => nil}` instead of `%{0 => 0}`, which is the desired behaviour.